common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -86,7 +104,7 @@ public ADALOAuth2TokenCache(final Context context) {
     }
 
     protected void initializeSharedPreferencesFileManager(final String fileName) {
-        mISharedPreferencesFileManager = new SharedPreferencesFileManager(super.mContext, fileName);
+        setISharedPreferencesFileManager(new SharedPreferencesFileManager(super.getContext(), fileName));
```

Comment:
Remove `getContext` method, declare as `final protected`

Reply:
Checkstyle won't let us do this way. See above.